new-readme.md

```diff
@@ -3,30 +3,31 @@
 
 [![Build Status](https://travis-ci.org/reimagined/resolve.svg?branch=master)](https://travis-ci.org/reimagined/resolve) [![npm version](https://badge.fury.io/js/create-resolve-app.svg)](https://badge.fury.io/js/create-resolve-app) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/reimagined/resolve/master/LICENSE)
 
-reSolve is a framework for developing apps based on **CQRS**, **Domain-Driven Design** and **Event Sourcing** using **React + Redux** on the client. It negates differences between your domain and technical languages, and allows you to focus on application code.
+A JavaScript framework for **CQRS**, **Domain-Driven Design** and **Event Sourcing** with good support for **React** and **Redux**. **reSolve** provides a new way to describe app's core logic and fully separate it from an auxiliary code.
```

Comment:
Is 'with good support for React' good phrase?
There are 100500 react-based libs but i cannot google some examples of this phrase.

Reply:
As far as I understand, we are not BASED on React and it's possible to use reSolve without React. Am I right?

Reply:
This does not matter, i talk about specific phrase only
(yes, you are right)

Reply:
This text anyway goes to the corrector next. If something's wrong, we'll know. Thinking about the word 'good', I also start to dislike it... Maybe 'well' ?